sky/provision/__init__.py

```diff
@@ -8,6 +8,7 @@
 import functools
 import importlib
 import inspect
+from sky import status_lib
```

Comment:
```suggestion

from sky import status_lib
```
nit: https://google.github.io/styleguide/pyguide.html#313-imports-formatting